lighthouse-core/test/results/sample_v2.json

```diff
@@ -2934,6 +2934,13 @@
       "score": null,
       "scoreDisplayMode": "notApplicable"
     },
+    "structured-data-automatic": {
+      "id": "structured-data-automatic",
+      "title": "JSON-LD structured data syntax is valid",
+      "description": "Structured data contains rich metadata about a web page. The data is used in search results and social sharing. Invalid metadata will affect how the page appears in these contexts. This audit currently validates a subset of JSON-LD rules. See also the manual audit below to learn how to validate other types of structured data.",
+      "score": null,
+      "scoreDisplayMode": "notApplicable"
```

Comment:
would be nice to update our artifacts sometime with all the new goodies you've worked on, then we'd get the report previews for free :) 

seems fine for a separate PR though 👍

Reply:
Why is updating the artifacts such a pain? Couldn't we just have script that reverts everything that's expected to vary between runs?

Reply:
> everything that's expected to vary between runs

The problem is that list is enormous and not always separable from what you wanted to change :) 

I think to your point though, there's room for a more targeted script that says "only accept these particular artifact changes" and throw away everything else. 👍